src/components/ui/SuggestsDropdown.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+import { object, func, string, arrayOf } from 'prop-types';
+
+import SuggestItem from './SuggestItem';
+
+const SuggestsDropdown = ({
+  className = '',
+  suggestItems,
+  onSelect,
+  onHighlight,
+}) => {
+  const [highlighted, setHighlighted] = useState(null);
+
+  useEffect(() => {
+    const keyDownHandler = ({ key }) => {
+      if (key === 'ArrowDown') {
+        const h = highlighted === null ? - 1 : highlighted;
+        if (h < suggestItems.length - 1) {
+          setHighlighted(h + 1);
+          onHighlight(suggestItems[h + 1]);
+        } else {
+          setHighlighted(null);
+          onHighlight(null);
+        }
+      }
+
+      if (key === 'ArrowUp') {
+        const h = highlighted === null ? suggestItems.length : highlighted;
+        if (h > 0) {
+          setHighlighted(h - 1);
+          onHighlight(suggestItems[h - 1]);
+        } else {
+          setHighlighted(null);
+          onHighlight(null);
+        }
+      }
+
+      if (key === 'Enter') {
+        if (highlighted !== null) {
+          onSelect(suggestItems[highlighted]);
+        }
+      }
+    };
```

Comment:
The `Esc` key also has to be managed to close the dropdown. Or maybe in the parent as it is the one controlling the drowdown display status?

Reply:
It is handled elsewhere, so I should have not changed the existing behavior

Reply:
nope, I lied, There was a case to handle. I fixed it thanks